references/video_classification/train.py

```diff
@@ -149,7 +155,12 @@ def main(args):
     print("Loading validation data")
     cache_path = _get_cache_path(valdir)
 
-    transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    if not args.weights:
+        transform_test = presets.VideoClassificationPresetEval((128, 171), (112, 112))
+    else:
+        fn = PM.video.__dict__[args.model]
+        weights = PM._api.get_weight(fn, args.weights)
```

Comment:
nit: using a private API here. We probably don't want to advertise private APIs in the references

Reply:
This is temporary and behind a feature switch (`--weights`). It's already recorded to clean up at #4652